command/bridge/common/params.go

```diff
@@ -160,21 +161,58 @@ func ExtractExitEventID(receipt *ethgo.Receipt) (*big.Int, error) {
 	return nil, errors.New("failed to find exit event log")
 }
 
+// ExtractChildTokenAddr extracts predicted deterministic child token address
+func ExtractChildTokenAddr(receipt *ethgo.Receipt, childChainMintable bool) (*types.Address, error) {
+	var (
+		l1TokenMapped contractsapi.TokenMappedEvent
+		l2TokenMapped contractsapi.L2MintableTokenMappedEvent
+	)
+
+	for _, log := range receipt.Logs {
+		if childChainMintable {
+			doesMatch, err := l2TokenMapped.ParseLog(log)
+			if err != nil {
+				return nil, err
+			}
+
+			if !doesMatch {
+				continue
+			}
+
+			return &l2TokenMapped.ChildToken, nil
+		} else {
+			doesMatch, err := l1TokenMapped.ParseLog(log)
+			if err != nil {
+				return nil, err
+			}
+
+			if !doesMatch {
+				continue
+			}
+
+			return &l1TokenMapped.ChildToken, nil
+		}
+	}
+
+	return nil, nil
+}
+
 type BridgeTxResult struct {
-	Sender       string     `json:"sender"`
-	Receivers    []string   `json:"receivers"`
-	ExitEventIDs []*big.Int `json:"exitEventIDs"`
-	Amounts      []string   `json:"amounts"`
-	TokenIDs     []string   `json:"tokenIds"`
-	BlockNumbers []uint64   `json:"blockNumbers"`
+	Sender         string         `json:"sender"`
+	Receivers      []string       `json:"receivers"`
+	ExitEventIDs   []*big.Int     `json:"exitEventIDs"`
+	Amounts        []string       `json:"amounts"`
+	TokenIDs       []string       `json:"tokenIds"`
+	BlockNumbers   []uint64       `json:"blockNumbers"`
+	ChildTokenAddr *types.Address `json:"childTokenAddr"`
 
 	Title string `json:"title"`
 }
 
 func (r *BridgeTxResult) GetOutput() string {
 	var buffer bytes.Buffer
 
-	vals := make([]string, 0, 5)
+	vals := make([]string, 0, 7)
```

Comment:
```suggestion
	vals := make([]string, 0, 6)
```

Reply:
In the worst case scenario it would be 7 (for ERC1155 which have amounts and token ids set). So I'd leave it to 7 if you agree?